README.md

```diff
@@ -8,7 +8,7 @@ Example:
 GET
 ```java
 HttpRequest request = httpRequest(new TargetUrl("abv.bg")).build();
-ChitaHttp chitaClient = new ChitaHttpClient();// in most cases chitaClient should be injected
+ChitaHttp chitaClient = new HttpClient();// in most cases chitaClient should be injected
```

Comment:
If we keep this interface I think  that it would be better if we make a fake object instead of mocking this one. Mocking with jmock/easymock/mockito will add complexity to the tests and production code will be replicated in tests as is. 

I looked at the test and what I see is the following: 

```java
    context.checking(new Expectations() {{
      oneOf(endpoints).userSessionsEndpoint("anyUsername");
      will(returnValue("http://myTestUrl/anyUsername"));

      oneOf(httpClient).execute(with(httpRequest));
      will(returnValue(stringHttpResponse("[{\"contractCode\":\"1234\",\"assignedIpAddress\":\"212.50.13.98\",\"macAddress\":\"00:0c:42:d0:6f:52\",\"userName\":\"pppoe-evo\",\"end\":null,\"start\":1377038382450}]")));
    }});
```

so when we change the chita API or some logic we have to change all tests, cause they are replicating most of the production code. In case we are using external http server we only have to change the code and the test will be same.